packages/zowe-explorer/src/job/ZoweJobNode.ts

```diff
@@ -211,7 +231,16 @@ export class Job extends ZoweTreeNode implements IZoweJobTreeNode {
         return this.getParent() ? this.getParent().getSessionNode() : this;
     }
 
+    set tooltip(newTooltip: string) {
+        if (newTooltip !== undefined) {
+            this._tooltip = newTooltip;
+        }
+    }
+
     get tooltip(): string {
+        if (this._tooltip !== undefined) {
+            return this._tooltip;
+        }
```

Comment:
Same as above

Reply:
addressed in commit https://github.com/zowe/vscode-extension-for-zowe/pull/1920/commits/1889233360272455accc66bbba7c913a16d6a1aa